tools/git_add_generated_dirs.sh

```diff
@@ -0,0 +1,2 @@
+BASEDIR=$(dirname "$0")
+(< $BASEDIR/generated_dirs.txt xargs -i find {} -type f) | xargs git add -f
```

Comment:
woooow I didn't know you could put the `<` before the command. Fancy shmancy.

For max portability, you should put a shebang on the shell script